HassDeviceClassesSourceGenerator/HomeAssistantTypesSourceGenerator.csproj

```diff
@@ -15,6 +15,14 @@
     <UserSecretsId>b54d135f-a346-4f3e-9dc3-705040d0e387</UserSecretsId>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;NU1803</NoWarn>
```

Comment:
NU1803 is warning that HTTP NuGet server access will be depreciated. A private NuGet server is used for testing.